packages/psss/src/utils/auth.js

```diff
@@ -5,22 +5,20 @@
 
 import { AccessPolicy } from '@tupaia/access-policy';
 
-export const checkIsAuthorisedForMultiCountry = user =>
-  getPermittedEntitiesForUser(user).length > 1;
+export const checkIsAuthorisedForMultiCountry = user => getEntitiesAllowedByUser(user).length > 1;
 
 export const checkIsAuthorisedForCountry = (user, match) =>
-  getPermittedEntitiesForUser(user).some(entityCode => entityCode === match.params.countryCode);
+  getEntitiesAllowedByUser(user).some(
+    entityCode => entityCode.toLowerCase() === match.params.countryCode,
+  );
 
-export const getPermittedEntitiesForUser = user => {
+export const getEntitiesAllowedByUser = user => {
   if (!user) {
     return [];
   }
 
-  // Todo: Update with the correct access policy check
-  // @see: https://github.com/beyondessential/tupaia-backlog/issues/1268
-  const accessPolicy = new AccessPolicy(user.accessPolicy);
-
-  // For testing...
-  return ['as', 'to'];
-  // return ['to'];
+  const entities = new AccessPolicy(user.accessPolicy).getEntitiesAllowed('PSSS');
```

Comment:
It'd be nice if we could parse the access policy once, and reuse the same `AccessPolicy` instance, for the sake of efficiency

Reply:
I think this comment no longer applies now since the access policy instantiation is now inside a cached selector.